src/seeds.proposals.cpp

```diff
@@ -225,11 +225,20 @@ void proposals::onperiod() {
         double majority = double(prop_majority) / 100.0;
         double fav = double(pitr->favour);
         bool passed = pitr->favour > 0 && fav >= double(pitr->favour + pitr->against) * majority;
-        bool valid_quorum = utils::is_valid_quorum(voters_number, quorum, total_eligible_voters);
         name prop_type = get_type(pitr->fund);
         bool is_alliance_type = prop_type == alliance_type;
         bool is_campaign_type = prop_type == campaign_type;
 
+        bool valid_quorum = false;
+
+        if (pitr->status != status_evaluate) {
```

Comment:
Sorry, I got confused.

I think the if statement is doing the opposite you described, right now, the quorum is immediately valid when the proposal is not in evaluate state and when the proposal is in evaluate state it is checking if the quorum is valid. I think this `pitr->status != status_evaluate` should be `pitr->status == status_evaluate`

Reply:
Thanks, you're right - that's why I need reviews ;)